updated Language docs for mobile, created one for Web (#766)
MapsPeople/docs: src/ios/v3/guides/display-language/index.md

<!-- Introduction -->
{% include "src/shared/display-language/intro.md" %}

> Addionally, aside from methods mentioned here, you can provide translations via the standard method for your device, such as `LocalizedString`.

Reviewer: ```suggestion
Additionally, aside from methods mentioned here, you can provide translations via the standard method for iOS, `LocalizedString`.
```
